doc/rados/operations/pools.rst

```diff
@@ -737,6 +737,105 @@ Managing pools that are flagged with ``--bulk``
 ===============================================
 See :ref:`managing_bulk_flagged_pools`.
 
+Setting values for a stretch pool
```

Comment:
We also need to update the stretch mode page to discuss making individual stretch pools, now that this is an option, and talk about when you would use which choices.

Reply:
Good idea thanks